docs/application-development/observability.md

```diff
@@ -58,67 +58,29 @@ For more information, see the [Go sample for metrics](https://github.com/tempora
 </TabItem>
 <TabItem value="java">
 
-To emit metrics in Java, you should use the following steps:
+To emit metrics with the Java SDK, use the [`MicrometerClientStatsReporter`](https://github.com/temporalio/sdk-java/blob/55ee7894aec427d7e384c3519732bdd61119961a/src/main/java/io/temporal/common/reporter/MicrometerClientStatsReporter.java#L34) implementation for Prometheus and other backends supported by [Micrometer](https://micrometer.io/docs), and use `WorkflowServiceStubsOptions.Builder.setMetricsScope` to set the metrics scope in your Worker or Client code.
```

Comment:
This doesn't look very clear for me in terms of structure.
The main problem is "MicrometerClientStatsReporter implementation for Prometheus ..."
Would be great if we restructure it to show more explicitly that JavaSDK has MicrometerClientStatsReporter adapter that provides an interface to Micrometer. Which, in turn, has adapters for specific metric backends.

Maybe something like:

To emit metrics with the Java SDK, use the `MicrometerClientStatsReporter` class to integrate with Micrometer `MeterRegistry` for your metric system. Micrometer *provides integration*(link here) with Prometheus and other backends.